src/pages/DownloadMonitor.vue

```diff
@@ -19,18 +19,45 @@
                         <div class="border-at-bottom pad pad--sides">
                             <div class="card is-shadowless">
                                 <p><strong>{{ downloadObject.initialMods.join(", ") }}</strong></p>
-                                <p v-if="downloadObject.progress < 100">Downloading: {{ downloadObject.modName }}</p>
-                                <p>{{Math.min(Math.floor(downloadObject.progress), 100)}}% complete</p>
-                                <Progress v-if="!downloadObject.failed"
-                                    :max='100'
-                                    :value='downloadObject.progress'
-                                    :className="['is-info']"
-                                />
-                                <Progress v-else-if="downloadObject.failed"
-                                    :max='100'
-                                    :value='100'
-                                    :className="['is-danger']"
-                                />
+
+                                <div class="row" v-if="!downloadObject.failed">
+
+                                    <div class="col">
+                                        <p v-if="downloadObject.downloadProgress < 100">Downloading: {{ downloadObject.modName }}</p>
+                                        <p v-else-if="downloadObject.installProgress < 100 && !downloadObject.failed">Downloading:</p>
```

Comment:
🐛 I think there's a bug somewhere, the `modName` never updates when downloading dependencies.

`&& !downloadObject.failed` is unnecessary as this `<div class="row">` is already conditional to it. Same thing a few rows later

Reply:
good catch, this never worked to begin with, but should be fixed now. I think one reason for this was that there were two different variables called `combo` so it was assumed they were the same :D